### PR TITLE
Formatting tweaks

### PR DESCRIPTION
**Pull request by clrux**
@talbs FYI

I like to display block-level elements as blocks and inline-level elements as inline. I think this helps reinforce what the element is and aids in scanning the markup to make changes. Plus it helps ensure that tags are closed properly.

The blockquote fix is here too. You didn't close the `</strong>`tag and the `blockquote` never closed as a result. Should be good to go now though!

Exciting.

### CODE REVIEW

**Comment by talbs** [comment]
Thanks, @clrux - that's a sample kitchen sink of HTML I copied/pasted from http://www.poormansstyleguide.com/. If the block-level formatting is off, its because I was lazy and 1) didn't check that the ``<strong>`` element was closed and 2) didn't think it was worth reformatting for the sake of just this proof of concept (of styling scope).

I agree that block level elements should be on their own lines, but don't agree with having closing tags of nodes that have only text being on their own lines. See below.

```html
<!-- nope -->
<div class="example">
    <p>Sed posuere consectetur est at lobortis. Cras mattis consectetur purus sit amet fermentum. 
       Sed posuere consectetur est at lobortis. Maecenas sed diam eget risus varius blandit sit amet non magna. 
       Nullam quis risus eget urna mollis ornare vel eu leo.
    </p>

    <p>
        Nulla vitae elit libero, a pharetra augue. Donec sed odio dui. Morbi leo risus, porta ac consectetur ac, vestibulum at eros. Nulla vitae elit libero, a pharetra augue.
    </p>
</div>

<!-- yep -->
<div class="example">
    <p>Sed posuere consectetur est at lobortis. Cras mattis consectetur purus sit amet fermentum. Sed posuere consectetur est at lobortis. Maecenas sed diam eget risus varius blandit sit amet non magna. Nullam quis risus eget urna mollis ornare vel eu leo.</p>p>

    <p>Nulla vitae elit libero, a pharetra augue. Donec sed odio dui. Morbi leo risus, porta ac consectetur ac, vestibulum at eros. Nulla vitae elit libero, a pharetra augue.</p>
</div>
```

If you want this to merge, please sync up the syntax to match that.

**Comment by clrux** [comment]
@talbs I unblocked the `p`'s but I kinda think it looks messy. Additional thoughts?

File: _posts/elements/2015-02-04-example-kitchen-sink.md [hunk 1]
```diff
@@ -341,30 +356,23 @@ description:  This is a kitchen sink of common HTML elements
...
+            <progress id="progress" value="65" max="100"></progress></p>
 
         <p><label for="meter">Meter:</label><br />
-            <meter id="meter" min="200" max="500" value="350">350 degrees</meter>
-        </p>
+            <meter id="meter" min="200" max="500" value="350">350 degrees</meter></p>
 
-        <p>
-            <button class="button">Button Element</button>
+        <p><button class="button">Button Element</button>
```
Comment (talbs) [inline _posts/elements/2015-02-04-example-kitchen-sink.md]: Messy Example: move nested ``button`` element to its own line. Right?

File: _posts/elements/2015-02-04-example-kitchen-sink.md [hunk 2]
```diff
@@ -267,71 +296,57 @@ description:  This is a kitchen sink of common HTML elements
...
-        </p>
+            <label><input type="radio" class="radio" name="radio_button" value="radio_4" disabled /> Radio Disabled</label><br/></p>
 
         <p>Checkboxes:<br />
             <label><input type="checkbox" class="checkbox" name="checkboxes" value="check_1" /> Checkbox 1</label><br/>
             <label><input type="checkbox" class="checkbox" name="checkboxes" value="check_2" /> Checkbox 2</label><br/>
             <label><input type="checkbox" class="checkbox" name="checkboxes" value="check_3" /> Checkbox 3</label><br/>
-            <label><input type="checkbox" class="checkbox" name="checkboxes" value="check_4" disabled /> Checkbox Disabled</label><br/>
-        </p>
+            <label><input type="checkbox" class="checkbox" name="checkboxes" value="check_4" disabled /> Checkbox Disabled</label><br/></p>
```
Comment (talbs) [inline _posts/elements/2015-02-04-example-kitchen-sink.md]: Messy Example: move nested elements to their own lines. Right?

**Comment by talbs** [comment]
Hey, thanks for unblocking things, @clrux . 

I agree that it looks a bit messy when ``<p>`` elements have other elements nested inside of them. **When a ``<p>`` or any other has nothing but text inside of it, my preference is to leave the text on the same line as the element**. If there are other HTML elements nested in something, they should be broken into their own line and indented properly. What do you think?

- - -

I think the other thing I'm being thrown off by is some line breaks in the code I think your Sublime Text settings are creating. Looks like you have a hard stop set for a certain character length. I'm all for it if you want to keep it, just do the following two things for me, please:

* tell me how to set it up locally in Sublime Text
* add a note in our general wiki settings saying we want a hard stop at X char length

Again, this is just a copy/paste job of a ton of base HTML elements to just start testing any styles against. I may not be something we 1) hang onto for long or 2) need to greatly clean up. If its really bothering you, I'm happy to remove it.

File: _posts/elements/2015-02-04-example-kitchen-sink.md (hunk 2 above)
Comment (clrux) [inline _posts/elements/2015-02-04-example-kitchen-sink.md]: Do you want `p`'s on their own line? I thought you didn't want that.

Comment (talbs) [inline _posts/elements/2015-02-04-example-kitchen-sink.md]: lol, sorry - I might be having a hard time reading this line/diff of code. Here's what I'd ideally like:

```html
<p>Checkboxes:<br />
    <label>
        <input type="checkbox" class="checkbox" name="checkboxes" value="check_1" />
        Checkbox 1
    </label><br/>
    <label>
        <input type="checkbox" class="checkbox" name="checkboxes" value="check_2" /> 
        Checkbox 2
    </label><br/>
    <label>
        <input type="checkbox" class="checkbox" name="checkboxes" value="check_3" /> 
        heckbox 3
    </label><br/>
    <label>
        <input type="checkbox" class="checkbox" name="checkboxes" value="check_4" disabled /> Checkbox Disabled
    </label><br/>
</p>
```

Again this is just copy/paste code from  a vendor, so ideally we wouldn't have the ``<br />`` elements in anything we've written.

**Comment by talbs** [comment]
:+1:

**Pull request merged**